funcs/func_groupcount.c

```diff
@@ -51,37 +56,307 @@
+			<para>If only the group is specified, then return group@category results that only match against the group name</para>
+			<para>If only the categoryis specified, then return group@category results that only match the against the category name</para>
+			<para>If the search parameters are entirely empty, all channels having any kind of group membership will be returned.</para>
+			<example>
+				Find channels that are members of the group 'foobarbaz':
+
+				On Channel SIP/1234-0000001:
+					Set(GROUP()=groupName)
+				On Channel SIP/1234-0000003:
+					Set(GROUP()=foobarbaz)
+				On Channel SIP/1234-0000004:
+					Set(GROUP()=foobarbill)
+				On Channel SIP/1234-0000005:
+					Set(GROUP()=somethingelse)
+				On any Channel:
+					Assuming that channels still exist that have the above associated GROUP() assignents
+					Set(channels_found=${GROUP_CHANNEL_LIST(foobarbaz)}
+				Given the above:
+					This will find the channels that are in the group 'foobarbaz'
+                                        Variable channels_found will be set to: SIP/1234-0000003
+			</example>
+			<example>
+				Find channels that have the category name of 'barbaz'
+
+				On Channel SIP/1234-0000001:
+					Set(GROUP()=groupName@categoryName)
+				On Channel SIP/1234-0000002:
+					Set(GROUP()=foo@barbaz)
+                                On Channel SIP/1234-0000003:
+					Set(GROUP()=oof@barbaz)
+                                On Channel SIP/1234-0000004:
+					Set(GROUP()=anything@somethingelse)
+				On any Channel:
+					Assuming that channels still exist that have the above associated GROUP() assignents
+					Set(channels_found=${GROUP_CHANNEL_LIST(@barbaz)})
+
+				Given the above:
+					This will find the channels that are in group 'foo@foobarbaz' and 'oof@barbaz' since they both have the category barbaz
+                                        Variable channels_found will be set to: SIP/1234-0000002,SIP/1234-0000003
+			</example>
+			<example>
+				Find groups containing 'foo' and category name of 'bar'
+
+                                On Channel SIP/1234-0000001:
+					Set(GROUP()=groupName@categoryName)
+                                On Channel SIP/1234-0000002:
+					Set(GROUP()=foo@bar)
+                                On Channel SIP/1234-0000003:
+					Set(GROUP()=foo@bar)
+                                On Channel SIP/1234-0000004:
+					Set(GROUP()=anything@somethingelse)
+				On any Channel:
+					Assuming that channels still exist that have the above associated GROUP() assignents
+					Set(channels_found=${GROUP_MATCH_CHANNEL_LIST(.*foo.*@.*bar.*)})
+
+				Given the above:
+					This will find the channels that are in the group@category assignment of foo@bar
+                                        Variable channels_found will be set to: SIP/1234-0000002,SIP/1234-0000003
+			</example>
+		</description>
+		<see-also>
+			<ref type="function">GROUP</ref>
+			<ref type="function">GROUP_MATCH_LIST</ref>
```

Comment:
This should be GROUP_MATCH_CHANNEL_LIST, not GROUP_MATCH_LIST, no, these are the similar ones? Or at least list both of them otherwise.

Reply:
Fixed